### PR TITLE
Prevent future dates in birthday

### LINKED ISSUE #196: [PE-D][Tester E] Bug in Upcoming Birthdays Feature Command
### Description

Upcoming Birthdays feature provides the contact details of contacts whose birthdays are in the next 60 days.

Calling `upcoming-b` shows some birthdays which have passed.

### Steps to Reproduce

1. `edit 8 b/30/03/2024`
2. `upcoming-b`

### Expected Output

List of contact details whose birthdays are in the next 60 days

### Actual Output

Testing done on : 31 March 2023

### Other details

Also tested birthdays on 30/03/2023 and 01/01/2024, but these did not show up in the list after calling `upcoming-b`.

Note that the birthdays can be put to the future (linked to [issue #6](https://github.com/joytqt-1202/ped/issues/6)). Not sure if that is causing some issue here.

Hope this helps with debugging!

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: joytqt-1202/ped#7

### PR DESCRIPTION
**Pull request by vsamarth**
Closes #196 and #177

### CODE REVIEW

**Comment by salty-flower** [comment]
Looks good, but maybe just tell the user "past but < 200 years":
image

File: src/main/java/seedu/connectus/model/person/Birthday.java [hunk 1]
```diff
@@ -14,6 +14,8 @@ public class Birthday {
     public static final String MESSAGE_CONSTRAINTS = "Birthday should be of the format DD/MM/YYYY\n"
             + "Format: " + PREFIX_BIRTHDAY + "BIRTHDAY";
 
+    public static final String MESSAGE_INVALID_DATE = "Birthday should be in the past";
```
Comment (salty-flower) [inline src/main/java/seedu/connectus/model/person/Birthday.java:17]: ...which points to here

**Review by salty-flower — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/connectus/model/person/Birthday.java (hunk 1 above)
Comment (vsamarth) [reply src/main/java/seedu/connectus/model/person/Birthday.java:17]: okay thanks for catching it

Comment (vsamarth) [reply src/main/java/seedu/connectus/model/person/Birthday.java]: i have pushed chages to fix it

**Review by salty-flower — APPROVED** [review]

**Pull request merged**